python/sglang/bench_serving.py

```diff
@@ -386,6 +389,118 @@ async def async_request_sglang_generate(
     return output
 
 
+async def async_request_sglang_grpc(
+    request_func_input: RequestFuncInput,
+    pbar: Optional[tqdm] = None,
+) -> RequestFuncOutput:
+    api_url = request_func_input.api_url
+    assert api_url.startswith("grpc://"), "gRPC URL must start with grpc://"
+
+    output = RequestFuncOutput()
+    output.prompt_len = request_func_input.prompt_len
+
+    try:
+        # Create gRPC request with same parameters as FastAPI
+        request = completion_pb2.CompletionRequest(
```

Comment:
https://github.com/sgl-project/sglang/blob/2125898af5224464f5b5999e32a6cc93f442199c/python/sglang/bench_serving.py#L315

This doesn't seem to match (e.g., return logprobs or  **request_func_input.extra_request_body,)